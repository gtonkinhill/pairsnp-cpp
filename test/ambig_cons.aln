>seq1
NGTCAGTC
>seq2
NG-CAGTC
>seq3
NGTGNGTA
>seq4
TGTTNG-C
>seq5
TGTTNG-C
