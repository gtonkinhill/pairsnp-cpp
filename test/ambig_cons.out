seq1	0	0	2	1	1
seq2	0	0	2	1	1
seq3	2	2	0	2	2
seq4	1	1	2	0	0
seq5	1	1	2	0	0
